### ISSUE TITLE
Different behaviour with or without snp.edges

### ISSUE THREAD

**Issue by PointerFLY** [SnapKit/SnapKit#291]
### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform                | OSX
 Platform Version        | 10.11.6
 SnapKit Version         | 0.30.0.beta2
 Integration Method      | CocoaPods
 
### Issue Description
When I am using SnapKit to constrain my view in an OSX app, I found:

```swift
let backgroundView = NSView()
self.view.addSubview(self.backgroundView)

// make backgroundView constrained to it's superview's edges

// It works
self.backgroundView.snp.makeConstraints {
   $0.edges.equalTo(self.view)
}

// It doesn't work
self.backgroundView.snp.makeConstraints {
   $0.edges.equalTo(self.view.snp.edges)
}
```
However, when I change to Version 0.22.0, the two snippets all works.

**Comment by robertjpayne** [comment]
@PointerFLY thanks for the bug report! Sounds fishy and likely due to a change I made so you could pin edges to margins. I'll double check on it.

**Comment by robertjpayne** [comment]
@PointerFLY fixed in `feature/0.40.0` branch head.

**Comment by PointerFLY** [comment]
Thank you, we appreciate you continuous effort aiming to make SnapKit a nice and clean DSL and a better substitute of tedious NSLayoutconstraint.
